fix(curriculum): moved `z-index` to new step (#49200)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/64076bbeba941114b0027b6a.md

---
id: 64076bbeba941114b0027b6a
title: Step 43
challengeType: 0
dashedName: step-43
---

# --description--

> Adding position `sticky` moved the element into its own stack, to ensure your `#years` don't get hidden by different stacks add `z-index` property set to `999` in `#years` CSS rule.

Reviewer: ```suggestion
Adding position `sticky` moved the element into its own stack. To ensure your `#years` element does not get hidden by different stacks, add a `z-index` property set to `999` in the `#years` rule.
```

# --hints--

Your `#years` selector should have a `z-index` property set to `999`.

```js

> assert(new __helpers.CSSHelp(document).getStyle('#years')?.getPropertyValue('z-index') === '999');

Reviewer: ```suggestion
assert.equal(new __helpers.CSSHelp(document).getStyle('#years')?.getPropertyValue('z-index'), '999');
```
